test/spec/EditorCommandHandlers-test.js

```diff
@@ -217,30 +211,45 @@ define(function (require, exports, module) {
             testToggleComment(expectedCommentedText, expectedCommentedSel, expectedSelText, "block");
         }
 
-        describe("Line comment/uncomment", function () {
-            beforeEach(setupFullEditor);
+        describe("Line comment/uncomment with indentLineComment enabled", function () {
+
+            beforeEach(function () {
```

Comment:
Sorry about being vague: I mean that there's extra indentation within this function block:
```
        beforeEach(function () {
                setupFullEditor();
                PreferencesManager.set("indentLineComment",true);
            });
```

Should be:

            beforeEach(function () {
                setupFullEditor();
                PreferencesManager.set("indentLineComment",true);
            });